packages/devextreme-scss/scss/widgets/generic/gridBase/_colors.scss

```diff
@@ -86,12 +86,6 @@ $datagrid-row-focused-bg: null !default;
 $datagrid-hover-bg: null !default;
 $datagrid-row-hovered-color: null !default;
 
-/**
-* $name 30. Context menu icon color
```

Comment:
We should not delete Theme Builder variables as them already became a part of public API. Project where these variables are used will be broken.